faker.go

```diff
@@ -1048,13 +1053,32 @@ func extractNumberFromText(text string) (int, error) {
 
 func randomString(n int, lang *langRuneBoundary) string {
 	b := make([]rune, 0)
+	set := make(map[rune]struct{})
+	if lang.exclude != nil {
+		for _, s := range lang.exclude {
+			set[s] = struct{}{}
+		}
+	}
+
+	counter := 0
 	for i := 0; i < n; {
 		randRune := rune(rand.Intn(int(lang.end-lang.start)) + int(lang.start))
+		for slice.ContainsRune(set, randRune) {
+			if counter++; counter >= maxGenerateStringRetries {
+				log.Fatal("Can't generate random sequence with exluded letters")
```

Comment:
Can we return an error here? 
Using fatal is quite dangerous, so I think, we just return the error, and let the user decide to do what they need to do, like calling fatal, or panic, or just log it?

Reply:
The error message is maybe something related to max-retries?

Reply:
Yes, sure. I will return an error and the error will indicate info about exhaustion of max-retries